index.html

```diff
@@ -37,7 +37,169 @@ <h1>I'm Newton <br>Glad to see you!</h1>
+                </div>
+            </li>
+            <li class="categories">
+                <img src="./img/portfolio-stories2.png" alt="Tonic Category Image">
+                <div class="left-block">
+                    <h2 class="project-title">
+                        Multi-Post Stories
+                    </h2>
+                    <span class="canopy">
+                        CANOPY
+                        <ul>
+                            <li><img src="./img/counter.png" alt=""></li>
+                            <li>Back End Dev</li>
+                            <li><img src="./img/counter.png" alt=""></li>
+                            <li>2015</li>
+                        </ul>
+                    </span>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="portfolio-tags">
+                        <li>
+                            html
+                        </li>
+                        <li>
+                            css
+                        </li>
+                        <li>
+                            javascript
+                        </li>
+                    </ul>
+                    <a href="" class="project-links">See Project</a>
+                </div>
+            </li>
+        </ul>
+    </section>
+    <section class="about-myself">
+        <h1>About Myself</h1>
+        <p >Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <span>
+            <h4 class="lets-connect">LET'S CONNECT</h4>
+        
+        <ul class="social-icons">
+            <li>
+                <a href="https://github.com/flemton" target="_blank"><img src="./img/github-icon.png" alt="GitHub Profile"></a>
+            </li>
+            <li>
+                <a href="https://www.linkedin.com/in/alhassan-o-83039a80/" target="_blank"><img src="./img/linkedin-icon.png" alt="linkedin Profile"></a>
+            </li>
+            <li>
+                <a href="https://web.facebook.com/n3wtongh/?_rdc=1&_rdr"><img src="./img/Vector.png" alt="Facebook Profile" width="18px" height="20px"></a>
+            </li>
+            <li>
+                <a href="https://twitter.com/oalhassan847" target="_blank"><img src="./img/twitter-icon.png" alt="linkedin Profile"></a>
+            </li>
+            <li>
+                <a href="https://medium.com/" target="_blank"><img src="./img/medium-icon.png" alt="Medium Page"></a>
+            </li>
+        </ul>
+        <a href="" class="project-links">Get my resume</a>
+        </span>
+        <ul class="skills-list">
+            <li>
+                <img src="./img/languages.png" alt="">
+            </li>
+            <li><img src="./img/languages-list.png" alt=""></li>
```

Comment:
:red_circle: As the previous reviewer said Using an image tag to represent this section is not a correct implementation. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list ul and each of them is a li tag. Please make sure your build this using HTML and CSS. The purpose here is to  show that you are able as a developer build a website using HTML and CSS  given the design

Read more about this discussion [Why do we make websites with code instead of just displaying an image of it?](https://www.quora.com/Why-do-we-make-websites-with-code-instead-of-just-displaying-an-image-of-it)


![share](https://user-images.githubusercontent.com/26736582/205434003-dadb7347-56c1-476a-a4af-5a756e630044.png)

![response](https://user-images.githubusercontent.com/26736582/205434012-c7683444-ed92-40b2-8e11-d1b57bda8684.png)